build.gradle

```diff
@@ -56,34 +56,31 @@ targetCompatibility = 1.8
 
 // keep in alphabetical order
 dependencies {
-    compile "com.google.code.gson:gson:${versions.gson}"
-    compile "com.google.guava:guava:${versions.guava}"
-    compile "com.google.inject:guice:${versions.guice}"
-    compile "com.typesafe:config:${versions.typesafeConfig}"
-    compile "com.typesafe.slick:slick_%%:${versions.slick}"
-    compile "com.typesafe.slick:slick-hikaricp_%%:${versions.slick}"
-    compile "commons-io:commons-io:${versions.commonsIo}"
+    implementation "com.google.code.gson:gson:${versions.gson}"
```

Comment:
`compile` and `runtime` have been deprecated in favor of `implementation`: 

https://docs.gradle.org/6.0/userguide/upgrading_version_5.html#changes_6.0

Reply:
👍